policy/modules/services/sasl.if

```diff
@@ -38,6 +38,24 @@ interface(`sasl_read_keytab',`
 	read_files_pattern($1, saslauthd_keytab_t, saslauthd_keytab_t)
 ')
 
+########################################
+## <summary>
+##	Memory map SASL keytab files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`sasl_map_keytab',`
```

Comment:
I'd prefer to get away from interfaces that only provide `map`, so in this case having a `sasl_mmap_read_keytab` would be better.